packages/bpk-component-phone-input/readme.md

```diff
@@ -0,0 +1,67 @@
+# bpk-component-phone-input
+
+> Backpack phone input component.
+
+## Installation
+
+```sh
+npm install bpk-component-phone-input --save-dev
+```
+
+## Usage
+
+```js
+import React, { Component } from 'react';
+import BpkPhoneInput from 'bpk-component-phone-input';
+
+export default class extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { dialingCode: '44', phoneNumber: '' }
+  }
+
+  onChange = (dialingCode, phoneNumber) => {
+    this.setState({ dialingCode, phoneNumber });
+  }
+
+  render() {
+    return (
+      <BpkPhoneInput
+        id="phone-input-id"
+        name="Telephone input"
+        placeholder="Telephone number"
+        onChange={this.onChange}
+        value={this.state.phoneNumber}
+        dialingCode={this.state.dialingCode}
+        dialingCodes={[
+          { code: '44', description: '+44' },
+          { code: '55', description: '+55' },
+        ]}
+        dialingCodeProps={{
+          id: 'dialing-code',
+          name: 'Dialing code',
+          'aria-label': 'Dialing code',
+        }}
+      />
+    )
+  }
+}
+```
+
+## Props
+
+| Property              | PropType                                              | Required   | Default Value    |
+| --------------------- | ----------------------------------------------------- | ---------- | ---------------- |
+| dialingCode           | string                                                | true       | -                |
+| dialingCodeProps      | shape({ id: string, name: string })                   | true       | -                |
```

Comment:
`id` and `name` are required and the rest is optional, similar to how we use`...rest`. `classname` is in the flow declaration because I'm using it directly and otherwise flow whines about it. Do you have any idea how to type it better? Maybe I can mention bellow that `id` and `name` are required but any other property can be passed and will be forwarded to the component.

Reply:
Yeah maybe mentioning it below is a good idea.

Things like this are making me think that we should re-think how we expose prop types to users..